content/en/docs/contribute/style/style-guide.md

```diff
@@ -113,7 +117,7 @@ The copy is called a "fork". | The copy is called a "fork."
 
 ## Inline code formatting
 
-### Use code style for inline code, commands, and API objects
+### Use code style for inline code, commands, and API objects {#code-style}
```

Comment:
nit: I would make the anchor text more specific. There are a couple of `code style` related sections after this section.
Possibly:
`{#code-style-inline-code-commands-api-objects}`